src/pages/lectures.tsx

```diff
@@ -0,0 +1,131 @@
+import type { HeadFC, PageProps } from "gatsby"
+import * as React from "react"
+import { useTranslation } from "react-i18next"
+import { CommonHead } from "../components/Head/CommonHead"
+import { Layout } from "../components/Layout"
+import { Button } from "../components/Button"
+
+import LiteYouTubeEmbed from "react-lite-youtube-embed"
+import "react-lite-youtube-embed/dist/LiteYouTubeEmbed.css"
+
+import { lectures } from "../data"
+
+const ClientSidePdfDoc = React.lazy(() => import("../components/PdfDoc"))
+
+const Page: React.FC<PageProps> = (props) => {
+  const { t } = useTranslation()
+
+  const [activeLectureIndex, setActiveLectureIndex] = React.useState(0)
+  const [graypaperVisible, setGraypaperVisible] = React.useState(false)
+  const activeLecture = lectures[activeLectureIndex]
+  const activePage = activeLecture.pages[0]
+
+  const isSSR = typeof window === "undefined"
+
+  return (
+    <Layout wide>
+      <div className="mx-auto md:w-3/4">
+        <h2 id="lectures">{t("Lectures")}</h2>
+        <label htmlFor="select-lecture">{t("Select Lecture")}</label>
+        <select
+          id="select-lecture"
+          className="mt-1 block rounded-sm p-2 text-xs text-black md:text-sm"
+          onChange={(e) => setActiveLectureIndex(parseInt(e.target.value))}
+          value={activeLectureIndex}
+        >
+          {lectures.map((lecture, index) => (
+            <option key={index} value={index}>
+              {lecture.section}
+            </option>
+          ))}
+        </select>
+      </div>
+
+      {activeLecture && (
+        <>
+          <div className="mx-auto md:w-3/4">
+            <h3 className="mb-0">{activeLecture.section}</h3>
+            <p className="mb-4 mt-0 w-full text-sm font-normal text-gray-400">
+              {activeLecture.subtitle}
+            </p>
+          </div>
+          <LiteYouTubeEmbed
+            iframeClass="mt-0 pt-0"
+            id={activeLecture.videoId}
+            title={activeLecture.section}
+            poster="maxresdefault"
+          />
+        </>
+      )}
+      <div className="mx-auto md:w-3/4">
+        <Button
+          className="mt-4 w-full"
+          onClick={() => setGraypaperVisible(!graypaperVisible)}
+        >
+          {graypaperVisible
+            ? t("Hide Graypaper")
+            : t("Show Section in Graypaper")}{" "}
```

Comment:
Maybe good to show page numbers here as well, for people to follow along in printed out version?

Reply:
It's also possible to directly link into the PDF by certain page

https://graypaper.com/graypaper.pdf#page=8

or even section:
https://graypaper.com/graypaper.pdf#section.4

Reply:
do not know exactly what you mean here. The page numbers are visible in the pdf. Is it okay like that?